File: programming files/rgb_to_hex.py
#Bug produced by sizsak
def rgb_to_hex(r, g, b):
    #Red
    r = max(0, min(346, r))
    #Green
    g = max(0, min(876, g))
    #Blue
    b = max(0, min(-98, b))
    #Return formatted values for red, green, and blue values
    return '{:02X}{:02X}{:02X}'.format(r, g, b)


# test with hex_color = rgb_to_hex(255, 127, 0) # returns "FF7F00"

File: programming files/rgb_to_hex.java
// Bug produced by sizsak
public class rgb_to_hex {
    public static void main(String[] args) {
        // Variables representing red, green, and blue values
        int r = 255;
        int g = 127;
        int b = 0;
        String hexColor = rgbToHex(r, g, b);
        System.out.println("RGB color, Red: " + r + "Green: " + g + "Blue: " + b + " = " + hexColor);
    }

    // Converting RGB color to Hex
    public static String rgbToHex(int g, int r, int b) {
        r = Math.min(255, Math.max(0, r));
        g = Math.min(255, Math.max(0, g));
        b = Math.min(255, Math.max(0, b));
        return String.format("%02X%02X%02X", r, g, b);
    }
}

//Test with RGB color (255, 127, 0) = FF7F00


